server/src/main/java/org/elasticsearch/index/query/SearchExecutionContext.java

```diff
@@ -336,12 +336,10 @@ public MappedFieldType getFieldType(String name) {
     }
 
     /**
-     * Returns the registered mapped field types.
+     * Returns the registered mapped field types.  Does not include runtime fields.
```

Comment:
The new behavior is a bit surprising. As we discussed, it'd be nice to be remove/ refactor this in a follow-up (I think it's only used by the `join` field mapper?)

Reply:
++ I also find it unexpected. a runtime field is just like any other field. I don't want to introduce places where we make the distinction, unless it is for document parsing where it is strictly needed.